src/com/owncloud/android/ui/activity/Preferences.java

```diff
@@ -237,6 +238,28 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
             
         }
 
+        final Preference pCacheSize = findPreference("pref_cache_size");
+        if (pCacheSize != null){
+            final SharedPreferences appPrefs =
+                    PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+            Long cacheSize = ThumbnailsCacheManager.getMaxSize();
+            pCacheSize.setSummary(cacheSize + " Mb");
+            pCacheSize.setOnPreferenceChangeListener(new OnPreferenceChangeListener() {
+                @Override
+                public boolean onPreferenceChange(Preference preference, Object newValue) {
+                    String temp = (String) newValue;
+                    Long size = Long.decode(temp);
+                    if (ThumbnailsCacheManager.setMaxSize(size)){
+                        appPrefs.edit().putString("pref_cache_size", size.toString());
+                        pCacheSize.setSummary(size + " Mb");
```

Comment:
You are right!
Mb is also official Megabit, which is not the same as MB = MegaByte...